afl/src/bin/cargo-afl.rs

```diff
@@ -224,11 +244,27 @@ where
     I: IntoIterator<Item = S>,
     S: AsRef<OsStr>,
 {
-    let cmd_path = common::afl_dir(None).join("bin").join(tool);
-    let mut cmd = Command::new(cmd_path);
-    cmd.args(args);
+    let cmd_path: PathBuf;
+    let mut cmd;
+    if tool.to_string() == "sudo".to_string() {
+        cmd_path = tool.into();
+        cmd = Command::new(cmd_path);
+        let target = common::afl_dir(None).join("bin").join("afl-system-config");
+        let mut vec: Vec<&OsString> = vec![];
+        let os_tool = OsString::from(tool);
+        let os_target = target.into_os_string();
+        vec.push(&os_tool);
+        vec.push(&os_target);
+        let arguments = vec.into_iter();
+        cmd.args(arguments);
+    } else {
+        cmd_path = common::afl_dir(None).join("bin").join(tool);
+        cmd = Command::new(cmd_path);
+        cmd.args(args);
+    }
+
     let status = cmd.status().unwrap();
-    #[cfg(target_os = "macos")]
```

Comment:
Why did you remove this? Can the `shmget` error message appear on systems besides macOS? (That's the only place I've known it to occur.)

But while we are at it, could you please update the help message so that it refers to `cargo afl system-config`?

Reply:
I removed it because shm_open can fail on Linux too. Or any other system config error that afl-fuzz complains about - which can all be fixed with afl-system-config.